src/devices/machine/pit8253.h

```diff
@@ -34,13 +34,15 @@ enum class pit_type
 {
 	I8254,
 	I8253,
-	FE2010
+	FE2010,
+	PS2			/* like 8253, but weird */
 };
 
 class pit_counter_device : public device_t
 {
 	friend class pit8253_device;
 	friend class pit8254_device;
+	friend class ps2_pit_device;
```

Comment:
If you’re making derived classes friends, that usually means you’re doing something wrong – it’s a lot like replacing `private` with `protected`.  Why exactly is it doing this already, and why do you need to friend your derived class?  Better to fix it than perpetuate it.

Reply:
The 8253-likes are not derived from the PIT counter.